### PR TITLE
Update Gromix-MPX runtime to 46

### PR DESCRIPTION
**Pull request by ghost**
- Update Gromix-MPX runtime to 46

### CODE REVIEW

**Comment by ghost** [comment]
@bk138 I've updated MultiVNC runtime to version 46 since runtime version 44 has reached end-of-life. Could you test and review it.

**Comment by bk138** [comment]
Will do, time permitting.

**Comment by bk138** [comment]
@yakushabb I think you should also update gnome-settings-daemon https://github.com/flathub/net.christianbeier.Gromit-MPX/blob/c0af7757ce3ca52216f950efb91b994b207e7582/net.christianbeier.Gromit-MPX.yml#L27 to be in sync with the runtime. See 2a516b4c96f5fd80a5c14936e195f0bf1aa29c9d for an older update.

**Comment by ghost** [comment]
@bk138 I've added a few folders to the cleanup process to reduce the package size. Could you test them. I'm not sure about removing the `/share/pixmaps` folder.

File: net.christianbeier.Gromit-MPX.yml [hunk 1]
```diff
@@ -13,6 +13,14 @@ finish-args:
   - --talk-name=ca.desrt.dconf
   - --env=DCONF_USER_CONFIG_DIR=.config/dconf
   - --env=GIO_EXTRA_MODULES=/app/lib/gio/modules/
+cleanup:
```
Comment (bk138) [inline net.christianbeier.Gromit-MPX.yml]: please don't move this around in this PR, maybe a separate one.

**Review by bk138 — CHANGES_REQUESTED** [review]

**Comment by bk138** [comment]
> `flatpak install --user https://dl.flathub.org/build-repo/109343/net.christianbeier.Gromit-MPX.flatpakref`

There's a regression https://github.com/bk138/gromit-mpx/issues/210 so I'm going to hold back on this until it's fixed.

**Pull request closed without merging**